dockers/docker-orchagent/orchagent.sh

```diff
@@ -86,6 +86,9 @@ if [[ x"${LOCALHOST_SUBTYPE}" == x"SmartSwitch" ]]; then
     else
         ORCHAGENT_ARGS+=" -q tcp://127.0.0.1:8100"
     fi
+# Enable ZMQ for DPU Appliance
+elif [[ x"${LOCALHOST_SUBTYPE}" == x"Appliance" ]]; then
```

Comment:
I don't see a subtype 'Appliance'. It should be 'SmartSwitch' for both npu and dpu. @Pterosaur

Reply:
`Appliance` is not needed. I removed the changes.